casper/src/main/scala/coop/rchain/casper/util/rholang/RegistrySigGen.scala

```diff
@@ -172,9 +166,15 @@ object RegistrySigGen {
     * `user` (public key) and `timestamp`.
     *
     */
-  def genIds(user: ByteString, timestamp: Long) = {
-    val seed = DeployData().withUser(user).withTimestamp(timestamp)
-    Blake2b512Random(DeployData.toByteArray(seed))
+  def generateUnforgeableNameId(user: Array[Byte], timestamp: Long) = {
```

Comment:
Is it OK to have commit messages like just "refactoring"?

I'd suggest "factor out generateUnforgeableNameId".

not critical.

Reply:
Agreed